src/main/java/hudson/plugins/performance/constraints/AbsoluteConstraint.java

```diff
@@ -0,0 +1,146 @@
+package hudson.plugins.performance.constraints;
+
+import java.lang.reflect.InvocationTargetException;
+import java.util.List;
+
+import hudson.AbortException;
+import hudson.Extension;
+import hudson.model.AbstractBuild;
+import hudson.plugins.performance.PerformanceBuildAction;
+import hudson.plugins.performance.PerformanceReport;
+import hudson.plugins.performance.UriReport;
+import hudson.util.FormValidation;
+
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.QueryParameter;
+
+/**
+ * Absolute Constraints compare the result of a new load test against some user defined values.
+ * 
+ * @author Rene Kugel
+ *
+ */
+public class AbsoluteConstraint extends AbstractConstraint {
+
+	@Extension
+	public static class DescriptorImpl extends ConstraintDescriptor {
+
+		@Override
+		public String getDisplayName() {
+			return "Absolute Constraint";
+		}
+
+		public FormValidation doCheckRelatedPerfReport(@QueryParameter String relatedPerfReport) {
+			if ("".equals(relatedPerfReport)) {
+				return FormValidation.error("This field must not be empty");
+			}
+			return FormValidation.ok();
+		}
+
+		public FormValidation doCheckTestCase(@QueryParameter String testCase) {
+			if ("".equals(testCase)) {
+				return FormValidation.error("This field must not be empty");
+			}
+			return FormValidation.ok();
+		}
+	}
+
+	/**
+	 * User defined absolute value which must not be exceeded
+	 */
+	private long value = 0;
+
+	@DataBoundConstructor
+	public AbsoluteConstraint(Metric meteredValue, Operator operator, String relatedPerfReport, Escalation escalationLevel, boolean success, TestCaseBlock testCaseBlock, long value) {
+		super(meteredValue, operator, relatedPerfReport, escalationLevel, success, testCaseBlock);
+		this.value = value;
+
+	}
+
+	/**
+	 * Cloning of a AbsoluteConstraint Note that this is not from the Interface Clonable
+	 * 
+	 * @return clone of this object
+	 */
+	public AbsoluteConstraint clone() {
+		AbsoluteConstraint clone = new AbsoluteConstraint(this.getMeteredValue(), this.getOperator(), this.getRelatedPerfReport(), this.getEscalationLevel(), this.getSuccess(), new TestCaseBlock(this
+				.getTestCaseBlock().getTestCase()), this.getValue());
+		return clone;
+	}
+
+	@Override
+	public ConstraintEvaluation evaluate(List<? extends AbstractBuild<?, ?>> builds) throws InvocationTargetException, IllegalArgumentException, AbortException {
+		if (builds.isEmpty()) {
+			throw new IllegalArgumentException("Performance: No builds found to evaluate!");
+		}
+		checkForDefectiveParams(builds); // @Ivan: want to talk about this
+		PerformanceReport pr = builds.get(0).getAction(PerformanceBuildAction.class).getPerformanceReportMap().getPerformanceReport(getRelatedPerfReport());
```

Comment:
A AbsoluteConstraints always refers to the "newest build" (which is the build that ran immediately before this code is executed). The newest build is always at index 0 of this list.